### DISCUSSION TITLE
Pimcore Academy: "Portal Engine"

### DISCUSSION THREAD

**Discussion by adi84900** [pimcore/pimcore#15331] [Q&A / Support] [answered]
Can someone help me with questions 2,3 & 4

![image](https://github.com/pimcore/pimcore/assets/32416123/82a176c5-c427-413c-8d1d-003b8a91b38e)

@fashxp @willers-pm @NiklasBr @rajesh17692
Could you please help here

**Comment by twin-elements** [comment, marked as answer]
1
- Pimcore Experience Portals are Pimcore Sites and configuration primarily takes place in Pimcore Documents.

2
- Data pools define the displayed content of the portal.
- Each data pool provides a grid and detail pages for the content.
- Workspaces define what actual elements should be displayed in the data pool.
- Data pools are configured in configuration documents.
- A portal can have multiple data pools.

3
- With Pimcore data objects

4.  
-Workspaces define what elements are accessible in a data pool.
-There are two levels of workspaces - data pool and user.
-User workspaces can further restrict data pool workspaces.
 
5. Search is powered by Elasticsearch
6.
-Adding additional Symfony services for extending default behavior.
-Overwriting Twig templates.
-Configuring a custom frontend build for overwriting frontend implementation parts.
-Providing custom area bricks for content pages.
-Configuring stylings in portal configuration.
